Lib/test/test_pdb.py

```diff
@@ -1448,6 +1447,40 @@ def test_readrc_homedir(self):
                 if save_home is not None:
                     os.environ["HOME"] = save_home
 
+    def test_read_pdbrc_with_ascii_encoding(self):
+        script = textwrap.dedent("""
+            import pdb; pdb.Pdb().set_trace()
+            print('hello')
+        """)
+        save_home = os.environ.pop('HOME', None)
+        try:
+            with support.temp_cwd():
+                with open('.pdbrc', 'w') as f:
+                    f.write("Fran\u00E7ais\n")
```

Comment:
It will not work on locales with encoding which does not allow to encode `\u00E7`. You need to find characters which are encodable with the locale encoding.

Also, it would be better to use more realistic example. Use a command which takes a non-ASCII argument and check that it works correctly.

Reply:
Now the build is passing. And I would like to keep this negative test case as it would, in utf-8 encoding gives "NameError: name 'Française' is not defined". Or do you still want me write the test you commented above?